Requirements for the HW plugin to integrate with AUTO (#17053)
openvinotoolkit/openvino: docs/IE_PLUGIN_DG/Intro.md

*  :doc:`Quantized networks <openvino_docs_ov_plugin_dg_quantized_models>`
*  :doc:`Low precision transformations <openvino_docs_OV_UG_lpt>` guide
*  :doc:`Writing OpenVINO™ transformations <openvino_docs_transformations>` guide

> *  `Integration with AUTO Plugin <https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/auto/docs/integration.md>`

Author: ```suggestion
*  :doc:`Integration with AUTO Plugin <https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/auto/docs/integration.md>`
```

> *  :doc:`Integration with AUTO Plugin <https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/auto/docs/integration.md>`

Author: ```suggestion
*  Requirements of `integration with AUTO Plugin <https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/auto/docs/integration.md>`
```

> *  Requirements of `integration with AUTO Plugin <https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/auto/docs/integration.md>`

Author: ```suggestion
*  `Integration with AUTO Plugin <https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/auto/docs/integration.md>`__
```
